File: Lowercase.py
# Eden Ghirmai, 2/18/14, www.codeeval.com
# Given a string, converts it into lowercase 

import sys
test_cases = open(sys.argv[1], 'r')
for test in test_cases:
    if test: 
    	print(test.lower())

test_cases.close()

File: Easy/Lowercase.java
// Eden Ghirmai, 1/19/2014, www.codeeval.com
// Given a string, converts it into lowercase

import java.util.*;
import java.io.*;

public class Lowercase {
    public static void main (String[] args) throws FileNotFoundException {    
        File file = new File(args[0]);
        Scanner words = new Scanner(file);
     
	 	while (words.hasNextLine()) {
	 		String line = words.nextLine(); 
			line = line.toLowerCase();
			System.out.println(line); 
        }
    }
}
